db/migrate/20221019091306_add_user_ref_to_comment.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] Just as suggested above this should point to the `author`, not `user`. Kindly fix that